cuegui/cuegui/Constants.py

```diff
@@ -32,7 +32,12 @@
 import opencue
 
 
-VERSION = "1.3.0"
+possible_version_path = os.path.join(os.path.abspath(os.path.join(__file__ ,"../../..")), 'VERSION.in')
+with open(possible_version_path) as fp:
```

Comment:
Wrap this `with` block in a `if os.path.exists` block. If the file doesn't exist, just set `VERSION` to `"1.3.0"`, the default value it has now.

Also -- no need for the new variable, just set `VERSION` directly. So the block would like:

```
if os.path.exists(possible_version_path):
  with open(possible_version_path) as fp:
    VERSION = fp.read().strip()
else:
  VERSION = "1.3.0"
```

(I haven't tested this code)